### ISSUE TITLE
bounds="html" to wrapped custom component cannot drag in X-axis.

### ISSUE THREAD

**Issue by garciawarneckee** [react-grid-layout/react-draggable#382]
### Scenario
I found in the issues how to apply react-draggable to a custom component, basically, wrapping your custom component in a div. When I applied bounds="html" to that div I cannot move the component more than 8px in the X-axis. I made an example code in a public code sandbox, which is the following:

[Codesandbox example](https://codesandbox.io/s/mm0kjrp709)

**What I'm expecting**: Apply drag-and-drop to a custom element and not letting it go further of the browser window.

**Comments**: if I apply drag-and-drop directly to the native button element without the div, the bounds work as expected. Playing around with it I found it only happens when I apply the div that wraps the element.

**Comment by garciawarneckee** [comment]
I found a workaround to this issue. 
The closer solution to the expected behavior is to set the width of the div that encloses the custom component to 85px and the height to 20px (I think that this are the measures of my component which is a button, maybe after this could be dynamic to fit whatever component wraps). I´m not an expert but I understood when debugging the lib's code, that the capacity to move in the X & Y axis is derived from calculus using the bound tag (in my case html tag) and the proper element to be handled (custom-div in the example). Setting div's width and height to a small value, let me has a big difference in that calculus which is the ratio where I can drag-and-drop the element enclosed in the wrapper div. 

I put "closest" because there is a little right margin in the element where you cannot drag and drop but is not representative.

I updated the codesandbox if in order to show the workaround working.  

I hope the lib could provide a native solution to this and this solution to that issue helps to make it even greater than it already is.
